docs/en/latest/control-api.md

```diff
@@ -21,13 +21,12 @@ title: Control API
 #
 -->
 
-The control API can be used to
+In Apache APISIX, the control API is used to:
 
-* expose APISIX internal state
-* control the behavior of a single isolate APISIX data plane
+* Check the internal state of APISIX.
```

Comment:
I believe `expose` is the right word here compared to check. If you consider it a bit to the negative side, then `interact` sounds better to me.